dashboard/test/ui/test_status.js

```diff
@@ -306,4 +307,5 @@ function toggleAutoRefresh() {
 refreshButton.onclick = refresh;
 autoRefreshButton.onclick = toggleAutoRefresh;
 enableAutoRefresh();
+updateProgress();
```

Comment:
Calling this on page load makes the progress bars show up immediately, where before they would be invisible until there was progress.